sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/factors/variables/TargetV.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sensitivity.factors.variables;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.sensitivity.SensitivityVariable;
+
+import java.util.Objects;
+
+/**
+ * Sensitivity variable on a voltage target at a regulating terminal
+ *
+ * @author Anne Tilloy {@literal <anne.tilloy at rte-france.com>}
+ */
+public class TargetV extends SensitivityVariable {
+
+    @JsonProperty("variableId")
+    private final String regulatingTerminalId;
+
+    /**
+     * Constructor
+     *
+     * @param id unique identifier of the variable
+     * @param name readable name of the variable
+     * @param regulatingTerminalId id of the network regulating terminal which targetV increase is used as sensitivity variable
+     * @throws NullPointerException if regulatingTerminalId is null
+     */
+    @JsonCreator
+    public TargetV(@JsonProperty("id") String id,
```

Comment:
Indeed I think it will be really better to use the `BusRef` here to and remove the regulating terminal approach.

Reply:
It's not consistent with what I proposed above :) we need to agree : I think it's simpler to have the ID of the regulating equipment here (see mail from Vincent).

Reply:
I think it becomes no simpler for other TSO networks as they rely a lot on shared and remote controls. And, furthermore, we will be obliged to look for the regulating bus inside the loadflow implementation. But, as they is no need for European process it is not a big deal on my side :-)

Reply:
Ok, what I don't know is : in that use case, will it be simpler for them to define the regulated bus, or one of the equipments which regulate that bus ? I would think the second is more natural, but not sure.

Anyway, in that case, it will be better to have both possibilities, so I propose to start with the regulating equipment ID and add another implementation when needed, is it OK?

Reply:
Yes.